tests/Plugins/GitHubModule/ClassicBranchProtectionRequirements/DismissStalePullRequestApprovals_UnitTest.py

```diff
@@ -32,44 +32,44 @@ def requirement():
     return DismissStalePullRequestApprovals()
 
 
+@pytest.fixture(name="requirement_args")
+def requirement_args_fixture():
+    return {"disabled": False}
+
+
 class TestDismissStalePullRequestApprovals:
-    def test_required_pull_request_reviews_missing(self, requirement, query_data):
+    def test_required_pull_request_reviews_missing(self, requirement, query_data, requirement_args):
```

Comment:
Why is it necessary to pass `requirement_args` with the default value?

Reply:
We are now checking the `requirement_args` as the first thing to evaluate the `__expected_value`. This is then used to make the description more informative.